### DISCUSSION TITLE
Get hierarchy API issue

### DISCUSSION THREAD

**Discussion by sumitbajaj-tarento** [Sunbird-Knowlg/Community#80] [Bugs]
Hi Sunbird Team,

We are trying to edit a LIVE course so calling **GET Hierarchy API** with mode=edit but I am getting internal server error while fetching the data. Without mode=edit or with hierarchyType=details its working fine. This is the endpoint which we are using for **getHierarchy**  {{host}}/apis/proxies/v8/action/content/v3/hierarchy/do_1134170689871134721450?mode=edit. We are using repo as **https://github.com/sunbird-cb/knowledge-platform.git** and **branch as cbrelease-4.0.1**. Attaching screenshot regarding error stack for the same. Please help us out regarding this issue.

![Screenshot (244)](https://user-images.githubusercontent.com/89627266/203018716-a4d2b706-86ac-4b6d-af97-c4b6c5fb7240.png)

**Comment by pkranga** [comment]
@vinukumar-vs request  your help in this

**Reply by Jayaprakash8887** [reply]
@pkranga Issue is because 'relational_metadata' column in hierarchy_store.Content_hierarchy table is returning null/inappropriate value. Make an empty json entry in the column (Ex: "{}") and check if the issue exists.

**Reply by sumitbajaj-tarento** [reply]
@Jayaprakash8887 We have checked the relational metadata column in Cassandra its showing as NULL value. We tried updating with {} but the issue remains same. We have checked in neo4j as well for the respective course ID and data exists there as well.

**Reply by Jayaprakash8887** [reply]
@sumitbajaj-tarento Is the value returned to the code from cassandra for the column 'relational_metadata' is still null even after updating it with '{}' ?
